unity/EditorPlugin/AssetPostprocessors/CsprojAssetPostprocessor.cs

```diff
@@ -149,10 +149,16 @@ private static bool UpgradeProjectFile(string projectFile, XDocument doc)
       changed |= SetXCodeDllReference("UnityEditor.iOS.Extensions.Common.dll", projectContentElement, xmlns);
       changed |= SetDisableHandlePackageFileConflicts(projectContentElement, xmlns); // already exists
       changed |= SetGenerateTargetFrameworkAttribute(projectContentElement, xmlns); // no need
+      changed |= CleanUpNoWarn(projectContentElement, xmlns); 
       
       return changed;
     }
 
+    private static bool CleanUpNoWarn(XElement projectContentElement, XNamespace xmlns)
+    {
+      return SetOrUpdateProperty(projectContentElement, xmlns, "NoWarn", existing => "");
```

Comment:
We should only remove `0169`, in case Unity add another warning in the future. I think they're semi-colon separated, but would have to check - it might support semi-colon and comma as a separator.

Reply:
This would not change. After 2019.2 all ProjectGeneration things are on the Rider package side. So I don't expect any surprises.

Reply:
Good point! 👍